rome.json

```diff
@@ -18,76 +18,99 @@
-        "useButtonType": "warn",
+        "noBlankTarget": "error",
+        "noDistractingElements": "error",
+        "noHeaderScope": "error",
+        "noPositiveTabindex": "error",
+        "useAltText": "error",
+        "useAnchorContent": "error",
+        "useButtonType": "error",
+        "useHtmlLang": "error",
         "useKeyWithClickEvents": "off",
-        "useKeyWithMouseEvents": "off",
+        "useKeyWithMouseEvents": "error",
         "useValidAnchor": "off"
       },
       "complexity": {
-        "noExtraBooleanCast": "warn",
-        "noMultipleSpacesInRegularExpressionLiterals": "warn",
-        "noUselessFragments": "warn",
-        "useFlatMap": "warn",
-        "useOptionalChain": "warn",
+        "noExtraBooleanCast": "error",
+        "noMultipleSpacesInRegularExpressionLiterals": "error",
+        "noUselessFragments": "error",
+        "useFlatMap": "error",
+        "useOptionalChain": "error",
         "useSimplifiedLogicExpression": "off"
       },
       "correctness": {
         "noChildrenProp": "off",
-        "noConstAssign": "warn",
-        "noEmptyPattern": "warn",
-        "noNewSymbol": "off",
-        "noRenderReturnValue": "warn",
+        "noConstAssign": "error",
+        "noConstructorReturn": "error",
+        "noEmptyPattern": "error",
+        "noNewSymbol": "error",
+        "noPrecisionLoss": "error",
+        "noRenderReturnValue": "error",
+        "noSetterReturn": "error",
+        "noStringCaseMismatch": "error",
         "noUndeclaredVariables": "off",
-        "noUnnecessaryContinue": "warn",
-        "noUnreachable": "warn",
+        "noUnnecessaryContinue": "error",
+        "noUnreachable": "error",
+        "noUnsafeFinally": "error",
         "noUnusedVariables": "off",
-        "noVoidElementsWithChildren": "warn",
-        "useValidForDirection": "warn"
+        "noVoidElementsWithChildren": "error",
+        "noVoidTypeReturn": "error",
+        "useValidForDirection": "error"
       },
       "performance": {
-        "noDelete": "warn"
+        "noDelete": "error"
       },
       "security": {
-        "noDangerouslySetInnerHtml": "warn",
-        "noDangerouslySetInnerHtmlWithChildren": "warn"
+        "noDangerouslySetInnerHtml": "error",
+        "noDangerouslySetInnerHtmlWithChildren": "error"
       },
       "style": {
-        "noArguments": "warn",
+        "noArguments": "error",
         "noImplicitBoolean": "off",
-        "noNegationElse": "off",
-        "noShoutyConstants": "warn",
-        "noUnusedTemplateLiteral": "warn",
+        "noNegationElse": "error",
+        "noNonNullAssertion": "off",
+        "noShoutyConstants": "error",
+        "noUnusedTemplateLiteral": "error",
+        "noVar": "error",
         "useBlockStatements": "off",
-        "useFragmentSyntax": "warn",
-        "useSelfClosingElements": "warn",
-        "useShorthandArrayType": "warn",
+        "useConst": "error",
+        "useDefaultParameterLast": "error",
```

Comment:
I think TS would enforce it, no? Does not hurt to add it though.

Reply:
TS does not enforce it, it's valid js, for example:

```ts
function test(a = 'a', b: string) {}
```
has the type
```ts
function test(a: string | undefined, b: string): void
```

Maybe we should keep this rule disabled actually, wdyt? 🤔
I know I have to disable it in our biggest repo.

Reply:
Ah, I thought the default parameters should be the last 🤔 May be I am confusing it with something else. I am okay with either. We can disable it for consistency with other repo if you prefer.

Reply:
May be I am confusing it with rest parameters 😄